test/integration/offline.js

```diff
@@ -13,6 +13,7 @@ PouchDB.plugin(require('pouchdb-adapter-memory'))
 // environment, so we declare it in global scope here.
 global.PouchDB = PouchDB
 global.pouchdbFind = pouchdbFind
+global.navigator = undefined
```

Comment:
I don't know why but test failed if I remove this line ;) https://travis-ci.org/cozy/cozy-client-js/builds/222804829#L640